spinner/build.sh

```diff
@@ -428,16 +424,16 @@ function check_usage {
 }
 
 function get_environment_descriptor {
-	local file="${SPINNER_LIFERAY_LXC_REPOSITORY_DIR}/automation/environment-descriptors/${LXC_ENVIRONMENT}.json"
+	local descriptor="${SPINNER_LIFERAY_LXC_REPOSITORY_DIR}/automation/environment-descriptors/${LXC_ENVIRONMENT}.json"
```

Comment:
@pedropaccola, can you please rename to `environment_descriptor` to keep the consistency with the method's name?

Thanks

Reply:
Will do! Thanks @natocesarrego